src/client/app/containers/UIOptionsContainer.js

```diff
@@ -23,7 +23,10 @@ function mapStateToProps(state) {
 function mapDispatchToProps(dispatch) {
 	return {
 		selectMeters: newSelectedMeterIDs => dispatch(changeSelectedMeters(newSelectedMeterIDs)),
-		fetchMetersDataIfNeeded: () => dispatch(fetchMetersDataIfNeeded())
+		changeDuration: barDuration => dispatch(changeBarDuration(barDuration)),
+		fetchMetersDataIfNeeded: () => dispatch(fetchMetersDataIfNeeded()),
+		changeChartType: chartType => dispatch(changeChartToRender(chartType)),
+		toggleBarStacking: () => dispatch(changeBarStacking())
```

Comment:
Fixed in next commit. I originally had it changed from changeBarStacking() to toggleBarStacking to differentiate the two functions in the file, which is unnecessary.